src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -76,6 +77,25 @@ export class MediaPlayerComponent {
     window.open("/song/awdklawj");
   }
 
+  setNewSong(song) {
+    const albumPromise = MusicService.getAlbumById(song.albumId);
```

Comment:
the data about artist and album should be passed as parameters of this method. You probably already have them in songs table

Reply:
Table components (ArtistSongsTable or AlbumSongsTable) are no ready yet, so I am not sure how these data will be implemented in these components.

Reply:
implement ONLY for mysongs table